jenkins.yml

```diff
@@ -1,3 +1,6 @@
+environment:
+  SELENIUM_VERSION: 2.52.0
```

Comment:
We now have the ability to define arbitrary environment variables in the YAML just like the fancy hosted services! Take that "so called" Circle CI 😝